lib/identifiers/isbn.rb

```diff
@@ -12,6 +12,10 @@ class ISBN
       \b
     /x
     ISBN_10_REGEXP = /
+      (?<!              # Do not accidentally match a hyphenated ISBN-13
```

Comment:
too late, I know. But could you update this comment since now also takes into account a spaced ISBN-13